index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Telnet</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <h2>“Hello World, we are software developers!” </h2>
+    <header id="header" class="container">
+        <div class="logo">
+            <img id="header-img" src ="https://upload.wikimedia.org/wikipedia/commons/b/b6/PuTTY_icon_128px.png">
+        </div>
+        
+
+     <nav id="nav-bar">
+        
+            <ul>
+                <a class="nav-link" href="#Plan">Plans</a>
+                <a class="nav-link" href="#Services">Services</a>
+                <a class="nav-link" href="#Premiun">Premiun Content</a>
+                <a class="nav-link" href="#Support">Support</a>
+            </ul>
+     </nav>
+
+
+    </header>
+    <section id="Plan"><center><h1>Plans</h1></center>
+        
+        
+
+        <div class="row">
```

Comment:
As I mentioned before, would be better not to leave too much spaces between code lines, and please make sure that you use a class instead of id.

Reply:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes 😎

Reply:
As I mentioned before, would be better not to leave too many spaces between code lines, and please make sure that you use a class instead of an id, it makes your code more clean, readable, and easy to understand.